packages/shared/src/components/post/write/CreatePostButton.tsx

```diff
@@ -74,11 +76,15 @@ export function CreatePostButton({
     >
       <Button
         {...buttonProps}
-        variant={sidebar ? ButtonVariant.Float : ButtonVariant.Secondary}
+        variant={
+          sidebar || footer ? ButtonVariant.Float : ButtonVariant.Secondary
+        }
         className={className}
         disabled={getIsDisabled()}
         icon={compact && <PlusIcon />}
-        size={isLaptop || sidebar ? ButtonSize.Medium : ButtonSize.Small}
+        size={
+          isLaptop || sidebar || footer ? ButtonSize.Medium : ButtonSize.Small
```

Comment:
Same as above here `With the above check where we return null if !footer, wouldn't this always be true?`